src/main/kotlin/ru/buhinder/alcopartyservice/repository/facade/InvitationLinkDaoFacade.kt

```diff
@@ -36,4 +37,8 @@ class InvitationLinkDaoFacade(
             .switchIfEmpty { invitationLinkId.toMono() }
     }
 
+    fun getAllByEventId(eventId: UUID): Flux<InvitationLinkEntity> {
```

Comment:
in buhinder **get** is used when the method throws an exception if nothing found and **find** is used when an object or mono empty is returned

Reply:
Ok, I think you d better to create some document on confluence for such things